static/app/components/profiling/profilingOnboardingSidebar.tsx

```diff
@@ -58,6 +90,20 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
 }
 
 function ProfilingOnboarding(props: CommonSidebarProps) {
+  return (
+    <TaskSidebar
+      orientation={props.orientation}
+      collapsed={props.collapsed}
+      hidePanel={() => {
+        props.hidePanel();
+      }}
```

Comment:
nit, but you can just pass the function reference
```suggestion
      hidePanel={props.hidePanel};
```